packages/patrol_cli/lib/src/features/test/ios_test_backend.dart

```diff
@@ -276,6 +268,31 @@ class IOSTestBackend extends TestBackend {
     }
   }
 
+  Future<String> _xcTestRunPath(bool real, String sdkVersion) async {
+    final Glob glob;
+    if (real) {
+      glob = Glob('Runner_iphoneos$sdkVersion-*.xctestrun');
+    } else {
+      glob = Glob('Runner_iphonesimulator$sdkVersion-*.xctestrun');
+    }
+
+    const suffix = 'build/ios_integ/Build/Products';
+    final root = join(_fs.currentDirectory.absolute.path, suffix);
+    _logger.detail('Looking for .xctestrun matching ${glob.pattern} at $root');
+    final files = await glob.listFileSystem(_fs, root: root).toList();
+    if (files.isEmpty) {
+      final cause = 'No .xctestrun file were found at $root';
+      throwToolExit(cause);
+    }
+
+    _logger.detail('Found ${files.length} match(es)');
+    for (final file in files) {
+      _logger.detail('Found ${file.absolute.path}');
+    }
+
+    return files.first.absolute.path;
```

Comment:
Maybe log something that you're using the first one found

Reply:
good idea!